docker/reverse_proxy/nginx-development.conf

```diff
@@ -1,25 +1,39 @@
+# Nginx config for routing within the docker network (development)
+
 upstream martin {
-    server martin:3000;
+    server martin:3000; # Host within docker network
 }
 
 server {
-    listen 80;
+    listen 80 default_server;
     server_name localhost;
 
-    location ~ /tiles/(?<fwd_path>.*) {
+    root /usr/share/nginx/html;
+
+    location = /map.html {
+    }
+
+    location = /style.json {
+    	add_header 'Access-Control-Allow-Origin' '*';
+
+    	sub_filter_once off;
+    	sub_filter_types application/json;
+        sub_filter "https://tiles.staging.ehrenamtskarte.app" "http://localhost:5002";
```

Comment:
is that right for development?

Reply:
Seems right. It gets replaced with localhost like before.